crates/chia-consensus/src/gen/get_puzzle_and_solution.rs

```diff
@@ -24,6 +25,7 @@ pub fn parse_coin_spend(
 pub fn get_puzzle_and_solution_for_coin(
     a: &Allocator,
     generator_result: NodePtr,
+    backrefs: &HashSet<NodePtr>,
     find_parent: Bytes32,
```

Comment:
Can we just pass in a `Coin` here, rather than the 3 individual fields?

Reply:
yes! I forgot that `Coin` is also a rust type now :)